lib/v2/njxzc/home.js

```diff
@@ -0,0 +1,101 @@
+// 导入got库，该库用来请求网页数据
+const got = require('@/utils/got');
+// 导入cheerio库，该库用来解析网页数据
+const cheerio = require('cheerio');
+// 导入parseDate函数，该函数用于日期处理
+const { parseDate } = require('@/utils/parse-date');
+// 导入timezone库，该库用于时区处理
+const timezone = require('@/utils/timezone');
+
+// 通知通告的url链接
+const url = 'https://www.njxzc.edu.cn/89/list.htm';
+// 南京晓庄学院的首页网址
+const host = 'https://www.njxzc.edu.cn';
+
+module.exports = async (ctx) => {
+    // 发起Http请求，获取网页数据
+    const response = await got({
+        method: 'get',
+        url,
+    });
+
+    // 解析网页数据
+    const $ = cheerio.load(response.body);
+
+    // 通知公告的items的url链接
+    const urlList = $('body')
+        .find('span.news_title a')
+        .map((i, e) => $(e).attr('href'))
+        .get();
+
+    // 通知公告的items的标题
+    const titleList = $('body')
+        .find('span.text a')
+        .map((i, e) => $(e).text())
+        .get();
+
+    // 通知公告的items的发布时间
+    const dateList = $('body')
+        .find('span.news_meta')
+        .map((i, e) => $(e).text())
+        .get();
```

Comment:
You can create single list instead of three lists
```js
    const list = $('.news_list .news')
        .toArray()
        .map((item) => {
            item = $(item);
            return {
                title: item.find('a').attr('title'),
                link: host + item.find('a').attr('href'),
                pubDate: timezone(parseDate(item.find('.news_meta').text(), 'YYYY-MM-DD'), +8),
            };
        });
```